Source/Plugins/Core/com.equella.core/scalasrc/com/tle/web/api/search/model/SearchParam.scala

```diff
@@ -95,4 +95,8 @@ class SearchParam {
     "List of search index key/value pairs to filter by. e.g. videothumb:true or realthumb:true.")
   @QueryParam("musts")
   var musts: Array[String] = _
+
+  @ApiParam("Custom Lucene query")
```

Comment:
```suggestion
  @ApiParam("Custom Lucene query which will be merged with the query generated from the other parameters")
```

Although in that suggestion where I put "will be merged" it would be good to say _how_ it will be merged. AND, OR, other?